src/main/java/frc/robot/subsystems/arm/ArmIOReal.java

```diff
@@ -123,235 +144,205 @@ public void setShoulderJointPower(double power) {
      * @param power desired power. [-1,1]
      */
     public void setElbowJointPower(double power) {
-        elbowMainMotor.set(TalonFXControlMode.PercentOutput, power);
+        elbowMainMotor.setControl(new DutyCycleOut(power));
     }
 
-    /**
-     * Gets the angle of the shoulder joint.
-     *
-     * @return angle of the shoulder joint. [rad]
-     */
-    public Rotation2d getShoulderJointAngle() {
-        return AngleUtil.normalize(Rotation2d.fromRotations(shoulderEncoder.getAbsolutePosition() - shoulderOffset));
-    }
+
 
     /**
      * Sets the angle of the shoulder joint.
      *
      * @param angle desired angle. [degrees]
      */
     public void setShoulderJointAngle(double angle) {
-        setShoulderJointAngle(angle, 0);
+        setShoulderJointAngle(angle, 0, 0);
     }
 
-    /**
-     * Sets the angle of the shoulder joint.
-     *
-     * @param angle desired angle. [degrees]
-     */
-    public void setShoulderJointAngle(double angle, double ffMultiplier) {
+
+    public void setShoulderJointAngle(double angle, double ffMultiplier, int velocity) {
```

Comment:
Why should external users be able to change the feedforward and velocity and controls?